src/test/java/qna/domain/entity/DeleteHistoryTest.java

```diff
@@ -18,23 +17,21 @@
 
 	@BeforeEach
 	void initialize() {
-		questionWrittenByJavajigi = new Question(1L, "title1", "contents1").writeBy(JAVAJIGI);
-		answerWrittenByJavajigi = new Answer(1L, JAVAJIGI, questionWrittenByJavajigi,
+		questionWrittenByJavajigi = Question.generate(1L, "title1", "contents1").writeBy(JAVAJIGI);
+		answerWrittenByJavajigi = Answer.generate(1L, JAVAJIGI, questionWrittenByJavajigi,
 			"Answers Contents1");
-		deleteHistoryOfQuestionWrittenByJavajigi = new DeleteHistory(ContentType.QUESTION,
-			questionWrittenByJavajigi.id(), questionWrittenByJavajigi.writer(),
-			LocalDateTime.now());
+		deleteHistoryOfQuestionWrittenByJavajigi = DeleteHistory.ofQuestion(
```

Comment:
변수명이 너무 길어지네요.
어차피 변수에 타입이 명시되어 있으니 어떤 변수인지 의미만 부여해보면 어떨까요

```suggestion
		자바지기_질문_삭제_이력 = DeleteHistory.ofQuestion(
```

Reply:
네이밍은 아직도 어려운 부분입니다 😢 
지금처럼 말씀하신대로
1. 타입은 명시가 되어 있으니 변수에는 타입을 명시하지 않도록 하고
2. 변수의 의미만 부여하여 읽기 좋게 해보자
라는 의미로 이해하겠습니다!
그리고 한글 변수명도 적극적으로 활용해 보겠습니다!!
👍 🙇